environment_dev.yml

```diff
@@ -4,7 +4,10 @@ channels:
 - defaults
 dependencies:
 - coverage=4.2=py36_0
+- jupyter=1.0.0=py36_7
```

Comment:
We specify that python=3.6.3, should this be =py36_7 or =py36_3?

Reply:
Good question. I copied this from the jupyter that got installed while using python3.6 and only specifying 1.0.0. Let's just take that off so whichever one can work.